RELEASE.md

```diff
@@ -227,8 +227,21 @@ Announce release!
   * enter the new version number as the tag prefixed with `v` e.g. (`v0.1.12`)
   * Make sure to include the links for diffs between the versions.
   * for release title choose a great name, no pressure
-  * in the description paste the upgrade instructions from the previous release,
-    followed by the new CHANGELOG. entry
+  * in the description paste the following upgrade instructions, filling in the version number for NEW_VERSION_NUMBER:
+
+```
+Setup
+
+`npm install -g ember-cli@NEW_VERSION_NUMBER` -- Install new global ember-cli
+
+Project Update
+
+1. `rm -rf node_modules dist tmp` -- Delete temporary development folders.
```

Comment:
I'm unsure if this is actually valuable any more? Do we really need to `rm -rf` these folders?